docker/Dockerfile.dmqnode

```diff
@@ -0,0 +1,26 @@
+# Duplicate definition of DIST before FROM is needed to be able to use it
+# in docker image name:
+ARG  DIST=xenial
+FROM sociomantictsunami/dlang:$DIST-v5 as builder
+# Copies the whole project as makd needs git history:
+COPY . /project/
+WORKDIR /project/
+# Redefine arguments as env vars to be used inside build.sh script:
+ARG DIST=xenial
```

Comment:
I can see a point of leaving this duplicate definition here for consistency. Is there any other reason why line 3 is not enough?

Reply:
I feel like I will be answering this question in every docker PR to be made :D 

It is necessary because `ARGS` defined before `FROM` only work until `FROM` and must be redefined to be kept.

Reply:
Ah, ok. Well you could make a comment for the most casual readers :-)